test/evaluation/uriEvaluationHandler.spec.ts

```diff
@@ -15,21 +15,21 @@ describe('evaluate', () => {
 
   it('should return ok if uris match in vpSimple.verifiableCredential[0].credentialSchema[0].id', () => {
     const pdSchema: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-simple-schema-age-predicate.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp-simple-age-predicate.json');
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.